optimize/self-managed/optimize-deployment/advanced-features/import-guide.md

```diff
@@ -129,16 +129,17 @@ The following image illustrates the components involved in the import process as
 
 During execution, the following steps are performed:
 
-1. [Start an import round](#start-an-import-round)
-2. [Prepare the import](#prepare-the-import)
-
-   - 2.1 Poll a new page
-   - 2.2 Map entities and add an import job
-
-3. [Execute the import](#execute-the-import)
-
-   - 3.1 Poll a job
-   - 3.2 Persist the new entities to Elasticsearch
+- [Architecture overview](#architecture-overview)
+- [Import performance overview](#import-performance-overview)
+  - [Setup](#setup)
+  - [Large size data set](#large-size-data-set)
+  - [Medium size data set](#medium-size-data-set)
+- [Import procedure](#import-procedure)
+  - [Start an import round](#start-an-import-round)
+  - [Prepare the import](#prepare-the-import)
+    - [Poll a new page](#poll-a-new-page)
+    - [Map entities and add an import job](#map-entities-and-add-an-import-job)
+  - [Execute the import](#execute-the-import)
```

Comment:
This looks like a surprising amount of changes for a renaming PR? Is this because it's out of date with main or something?

Reply:
Nope. I'm actually not sure how this was auto-updated like a ToC. Is this a Prettier glitch? I'm trying to update it to

1. [Start an import round](#start-an-import-round)
2. [Prepare the import](#prepare-the-import)
   1. Poll a new page
   2. 2.2 Map entities and add an import job
3. [Execute the import](#execute-the-import)
   1. 3.1 Poll a job
   2. 3.2 Persist the new entities to Elasticsearch

But each time I save it updates to a ToC.

Reply:
I just had to add periods next to the links. This fixed it.